code/testsuite/csheets/pf_Paladin.xml

```diff
@@ -2346,7 +2346,7 @@ BIO
 		</item>
 		<total>
 			<weight>98 lbs.</weight>
-			<value>7563.45 gp</value>
+			<value>7538.45 gp</value>
```

Comment:
Value changes?

Reply:
The amount off is the value of a silver holy symbol (25 gp). I was attempting to reduce the holy symbol to a single mod-required item, but the migration file doesn't allow EQMODs to be added. I thought I had fixed it, but apparently not.